tests/common_test.go

```diff
@@ -113,6 +131,24 @@ func (s *BaseTestSuite) resultOrError(res interface{}, err error) bool {
 	return true
 }
 
+func (s *BaseTestSuite) resultsOrError(res interface{}, err error) bool {
```

Comment:
What is this new function used for, exactly? Is it necessary?

Reply:
It's not necessary at all.
This function is for writing a test for FindAll() with with the same way of FindOne() tests.
I will remove this function.